bin/lint-staged.js

```diff
@@ -17,7 +19,8 @@ if (supportsColor.stdout) {
 // Do not terminate main Listr process on SIGINT
 process.on('SIGINT', () => {})
 
-const packageJson = JSON.parse(fs.readFileSync('package.json'))
+const packageJsonPath = path.join(fileURLToPath(import.meta.url), '../../package.json')
```

Comment:
Shouldn't this be:

```suggestion
const packageJsonPath = path.join(fileURLToPath(import.meta.url), 'package.json')
```

To get the correct `package.json` of _lint-staged_ itself?

Reply:
`fileURLToPath(import.meta.url)` returns the file path to the current script file (bin/lint-staged.js). The '../..' changes the path to be the root of the lint-staged package (where lint-staged's pacakge.json file is).

Reply:
Never mind, tested this locally and indeed `../../package.json` is correct

Reply:
Yes, thanks @coalman!